lms/models/course.py

```diff
@@ -33,3 +34,17 @@ class Course(BASE):
         server_default=sa.text("'{}'::jsonb"),
         nullable=False,
     )
+
+
+class _Course(Grouping):
+    __tablename__ = "course_grouping"
+    __mapper_args__ = {"polymorphic_identity": "course"}
```

Comment:
Extends grouping adding new columns in a new table that would be joined to `grouping` 

https://docs.sqlalchemy.org/en/14/orm/inheritance.html#joined-table-inheritance